.changeset/nasty-pants-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': minor
+---
+
+Update code of conduct and contributions guide
```

Comment:
This change doesn't quite often affect the library itself, right? I mean, this literally won't change anything in the published files. Perhaps we don't need a changeset for this one?
```suggestion
---
---
```